clusterman/draining/queue.py

```diff
@@ -357,18 +372,24 @@ def host_from_instance_id(
 
 def process_queues(cluster_name: str) -> None:
     draining_client = DrainingClient(cluster_name)
-    mesos_master_url = staticconf.read_string(f'clusters.{cluster_name}.mesos_master_fqdn')
-    mesos_secret_path = staticconf.read_string(f'mesos.mesos_agent_secret_path', default=None)
-    operator_client = operator_api(mesos_master_url, mesos_secret_path)
+    cluster_manager_name = staticconf.read_string(f'clusters.{cluster_name}.cluster_manager')
+    mesos_operator_client = None
+    kube_operator_client = KubernetesClusterConnector(cluster_name, None)
```

Comment:
Will this cause problems for mesos-specific drainer? I think we can do something like:
try:
   setup operator client
except Errors:
   error handling

for both of kubernetes and mesos coperator clients.